src/outfit.c

```diff
@@ -1067,6 +1168,9 @@ static void outfit_parseSBolt( Outfit* temp, const xmlNodePtr parent )
             temp->u.blt.spin = atof( buf );
             free(buf);
          }
+         /* Load the collision polygon. */
+         buf = xml_get(node);
+         outfit_loadPLG( temp, buf, 1 );
```

Comment:
Should warn if no polygon is available and give a hint or something on how to generate it.